src/languages/inform7.js

```diff
@@ -37,6 +37,7 @@ function(hljs) {
       },
       {
         className: 'section',
+        relevance: 8,
```

Comment:
We usually try to avoid very precise relevance boosters. If this required, just use 10.